Merchant/App/PaymentInput/PaymentInputViewController.swift

```diff
@@ -11,26 +11,26 @@ import BDCKit
 
 class PaymentInputViewController: PinViewController {
     
-    var amountLabel: BDCLabel = {
+    let amountLabel: BDCLabel = {
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possible many more places than just these...

Reply:
Yes, but you change it from `var` to `let`, please add the `fileprivate` in the same time.

Reply:
Well, as I mentioned this commit was only about correcting the value types not the access level. If I was going to correct the access level I would need to touch other places too, so essentially I would be doing 2 different things in one commit. If you would ask me then to revert either of them (type or the access level) that would not be so easy. That's why I haven't done it all in 1 time. I try to isolate the changes. 🙂

Reply:
You spent time to check every single variable to change it as a constant. So I am sure you have time to change the access level 😄

Reply:
- I didn't check every single variable I just simply used the Xcode search which filtered that for me 😉
- It's not about having or not having time. It was about not doing 5 different things in 1 commit.